### DISCUSSION TITLE
LH system Geometry estimate failed! V1

### DISCUSSION THREAD

**Discussion by Vejay-KS** [bitcraze/discussions#1083] [Q&A] [answered]
Hi Everyone!

I am new to CrazyFlie drones and I am trying to estimate the geometry using lighthouse deck and two V1 base stations configured as "b" and "c". I followed the steps in the getting started with lighthouse tutorial, but as the step 5 of Estimate Geometry process, I am getting "Geometry estimate failed!" error.

I tried estimating the geometry many times but still got the same error. I am running 2023.11 firmware version.
On powering the drone, the lighthouse deck shows red and blue light for couple of seconds, then orange light also shows. Then red is off and blue is stable and orange is blinking. 

Is there any way to check why this error occurs? Any suggestions on what can be done to resolve this?

Thanks in Advance!

Image for reference:
<img width="560" alt="image" src="https://github.com/bitcraze/discussions/assets/125238927/0c534745-40e7-4f8b-ada1-033146c3b770">

**Comment by Vejay-KS** [comment]
Additionally I have tried the method posted in https://github.com/orgs/bitcraze/discussions/118.

I noticed that I was getting similar messages including 
"Solution did not converge, it might not be good!"

I also plotted the parameters as suggested in the same discussion.

<img width="1089" alt="image" src="https://github.com/bitcraze/discussions/assets/125238927/4c48b75e-e089-4c99-b757-cc6a757ad53e">

I would like to know what does "converge" here means and how can I achieve it?

**Reply by knmcguire** [reply]
converge in this context means, that the geometry estimation algorithm is able to find a solution of where the basestations are.

**Comment by knmcguire** [comment]
Hi! I've just double checked it on my PC with the Lighthouse V1 basestations and it seem to work for me. We do use the sync cable though so our basestations are configured as A and b.

What is to approximate orientation of your basestations? When you were going through the steps of the wizard, was it able in all cases to get values from both basestations?

**Reply by Vejay-KS** [reply, marked as answer]
Hi!
Thank you for the reply. 
I found out the problem. Turns out, only 3 out of 4 of the sensors were working on the lighthouse board when I looked into the console during startup of crazyflie. 
I then used a new lighthouse board that had all 4 sensors working properly and I was able to estimate the geometry.

This discussion can be closed.

**Reply by knmcguire** [reply]
Thanks for letting us know! I've selected yours as an answer.
